.gitattributes

```diff
@@ -20,3 +20,5 @@ app/_locales/** linguist-generated
 # yarn berry suggested .gitattributes
 /.yarn/releases/** binary
 /.yarn/plugins/** binary
+
+app/*.html linguist-language=EJS
```

Comment:
This tells the GitHub UI to highlight our `app/*.html` files as if they were `EJS` template files (`eta` uses `ejs` syntax).